Issues when running locally from node_modules.

```
/node_modules/.bin/polyserve
module.js:341
    throw err;
    ^

Error: Cannot find module '/Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/polyserve/lib/polyserve.js/lib/cli'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/polyserve/bin/polyserve:20:13
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:44:21
    at ondir (/Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:187:31)
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:153:39
    at onex (/Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:93:22)
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:24:18
```


Also whats up with the issues? PolymerLabs needs to fix these issues, this is pretty ridiculous.

--- a/bin/polyserve
+++ b/bin/polyserve
@@ -12,12 +12,13 @@
 'use strict';
 
 var resolve = require('resolve');
+var path = require('path');
 
 process.title = 'polyserve';
 
-resolve('polyserve', {basedir: process.cwd()}, function(error, path) {
-  path = path || '..';
-  var cli = require(path + '/lib/cli')
+resolve('polyserve', {basedir: process.cwd()}, function(error, libpath) {
+  var clipath = libpath ? path.resolve(libpath, '../cli') : '../lib/cli';
+  var cli = require(clipath);
   cli.run(process.argv)
     .catch(function(e) {
       process.exit(69);

--- a/package.json
+++ b/package.json
@@ -18,6 +18,7 @@
     "polyserve"
   ],
   "dependencies": {
+    "command-line-args": "^2.1.6",
     "express": "^4.8.5",
     "find-port": "^1.0.1",
     "opn": "^3.0.2",

